pt/lessons/advanced/escripts.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: page
+title: Executáveis
+category: advanced
+order: 3
+lang: pt
+---
+
+Para criar executáveis em Elixir nós utilizaremos escript. Escript produz um executável que pode rodar em qualquer sistema que tenha Erlang instalado.
+
+## Sumário
+- [Começando](#comecando)
+- [Fazendo Parsing dos Argumentos](#fazendo-parsing-dos-argumentos)
```

Comment:
Sorry, I forgot about this one. If the title doesn't have accentuation, you don't need to create an anchor for that in the title.